modules/policy/src/main/java/org/eclipse/xpanse/modules/policy/policyman/PolicyManager.java

```diff
@@ -211,7 +190,7 @@ public void deletePolicy(UUID id) {
         policyStorage.deletePolicyById(id);
     }
 
-    private void checkPolicyWhetherCreated(Csp csp, String policy) {
+    private void checkPolicyDuplicate(Csp csp, String policy) {
```

Comment:
method name can be - `checkIfPolicyIsDuplicate`

Reply:
OK